src/core/templatetags/debug_tools.py

```diff
@@ -0,0 +1,32 @@
+from django import template
+from utils.logger import get_logger
+
+logger = get_logger(__name__)
+
+register = template.Library()
+
+
+class TraceNode(template.Node):
+    """
+    Allows you to set a trace inside a template.
+    Usage:
+    {% load debug_tools %}
+    ...
+    {% set_trace %}
+    """
+
+    def render(self, context):
+        try:
+            from nose import tools
+            tools.set_trace()         # Debugger will stop here
+        except ImportError:
+            import pdb
+            pdb.set_trace()           # Debugger will stop here
+        except ImportError:
```

Comment:
This would need to live within a nested try/except block, otherwise it will not be handled.

```
try:
    foo()
except Exception:
    try:
        bar()
    except Exception:
        pass
```

That said, `pdb` is part of the standard library, so there is no need to handle an import error here, so you can remove the second exception handler

Reply:
Thanks for catching that. Will remove it.